backend/app/core/finetune/evaluation.py

```
@@ -0,0 +1,241 @@
+import difflib
+import time
+import logging
+from typing import Set
```

Ruff (0.12.2)

`typing.Set` is deprecated, use `set` instead

(UP035)